components/modal/demo/async.md

```diff
@@ -63,7 +63,7 @@ class App extends React.Component {
           confirmLoading={confirmLoading}
           onCancel={this.handleCancel}
         >
-          <p>{ModalText}</p>
+          <p>{this.state.ModalText}</p>
```

Comment:
Look 53 line

Reply:
@xrkffgg  haha sorry

Reply:
> @xrkffgg haha sorry
what's wrong then? (https://ant.design/components/modal/)

Reply:
What‘s the problem?

Reply:
> What‘s the problem?

No problem. I thought "Open Modal with async logic" example's modal text is not 'Content of the modal' first time it is open. I just refreshed the page and it is fine. Sorry to bother. thanks for collab

Reply:
you may close this pr